README.md

```diff
@@ -1 +1,51 @@
-# IdolGAN
\ No newline at end of file
+# IdolGAN ~~[adoːkẽꜜɴ]~~
+
```

Comment:
Need more instruction of IdolGAN project. Like the purpose of this project, motivation, what we did and our further works. This abstract is so simple for IdolGAN main repository. Edit this pls.